docs/arcaflow/index.md

```diff
@@ -45,13 +46,13 @@ You need to pick the right tool for the job. Sometimes, you need something simpl
     - Wide range of available plugins
     - Simple workflows are easy to write
     - It runs comands over SSH by default
-    - Users can define their own connection plugins
     
     **What it does worse than Arcaflow**
     
-    - Parallelization is hard, it is intended for resource creation rather than generic workflows
-    - It does not have a typing system and cannot guarantee type-safety or validation of data
-    - It is written in Python, so either needs to be installed using system python dependencies, run in a container, or in a virtual environment.
+    - Ansible's approach to parallelization is in terms of executing the same tasks against different hosts in parallel (see "forks" and "strategy"). Defining different tasks to perform in parallel is more challenging.
+    - By default, input/output data is not strongly typed. Any data type validation must be done explicitly within playbooks.
+    - It is written in Python, so the control host where it is run from needs system python dependencies (or to run from inside a container or python virtualenv)
+    - Depending on the module used, there may be system requirements for python or other dependencies on the target host/container. See [ansible documentation](https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-handle-not-having-a-python-interpreter-at-usr-bin-python-on-a-remote-machine)
```

Comment:
I think I am fine with the content changes, but my nit is that this strays from the style of the document. All of the other comparison bullet points are succinct one-liners, and these are sentences with qualifications and references. I wonder if we can address the factual and tonal concerns here without the need for defensive posture.

Reply:
```suggestion
    - Depending on the module used, there may be system requirements for python or other dependencies on the target host/container.
        -  See [ansible documentation](https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-handle-not-having-a-python-interpreter-at-usr-bin-python-on-a-remote-machine)
```